dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -749,6 +752,379 @@ def test_min_max_id_properly_update(self, *mocks):
+
+    @classmethod
+    def setUp(cls):
+        test_utils.set_seed(seed=0)
+
+    @classmethod
+    def setUpClass(cls):
+        test_utils.set_seed(0)
+        cls.maxDiff = None
+        cls.input_data = [
+            'edited 9 hours ago',
+            '6. Do not duplicate code.',
+            '\t',
+            'Just want to caution against following this too rigidly.',
+            '\t',
+            '   ',
+            'When you try to DRY them up into a single generic abstraction, '
+            'you have inadvertently coupled those two business rules together.',
+            '   ',
+            '   ',
+            'Removing duplication that repeats the handling of the exact same '
+            'business rule is also usually a win.',
+            '',
+            'Duplicate words: business, win, code',
+            '\n\r',
+            'Reply',
+            'Share',
+            'Report',
+        ]
+        cls.dataset = pd.DataFrame(cls.input_data)
+
+        # turn off data labeler because if model changes, results also change
+        profiler_options = ProfilerOptions()
+        profiler_options.set({'data_labeler.is_enabled': False})
+
+        cls.profiler = UnstructuredProfiler(
+            cls.dataset, len(cls.dataset), options=profiler_options)
+        cls.profiler2 = UnstructuredProfiler(
+            pd.DataFrame(['extra', '\n', 'test\n', 'data .', 'For merging.']),
+            options=profiler_options
+        )
+        cls.report = cls.profiler.report()
+
+    def test_sample(self):
+        self.maxDiff = None
+        self.assertCountEqual(
+            ['Report',
+             'Reply',
+             'Removing duplication that repeats the handling of the exact same '
+             'business rule is also usually a win.',
+             'edited 9 hours ago',
+             'Just want to caution against following this too rigidly.'],
+            self.profiler.sample
+        )
+
+    def test_total_samples(self):
+        self.assertEqual(16, self.profiler.total_samples)
+
+    def test_empty_line_count(self):
+        self.assertEqual(7, self.profiler._empty_line_count)
+
+    def test_text_profiler_results(self):
+        # pop out times
+        self.assertIsNotNone(
+            self.report['data_stats']['statistics'].pop('times'))
+
+        # vocab order doesn't matter
+        expected_vocab = ['x', 'i', 'y', 's', '9', ',', 'u', 'b', 'f', 'Y', 'J',
+                          'v', 'r', 'o', 'a', '6', 'n', 'h', ' ', 'g', 'R', 't',
+                          'W', '.', 'm', 'c', 'l', 'e', 'p', 'w', 'S', 'd', 'D',
+                          ':']
+        self.assertCountEqual(
+            expected_vocab,
+            self.report['data_stats']['statistics'].pop('vocab'))
+
+        # vocab order doesn't matter, case insensitive, remove stop words
+        # assuming if words are correct, rest of TextProfiler is merged properly
+        # vocab order doesn't matter, case insensitive, remove stop words
```

Comment:
this line is the same with the above, remove?

Reply:
done